### ISSUE TITLE
RMC vs 3D models

### ISSUE THREAD

**Issue by AaronGulman** [ladunjexa/reactjs18-3d-portfolio#38]
Hi , 
Just quickly want to point out that without adding `enablePan={false}` to `OrbitControls`  of your 3D models users can hold the right mouse button (RMB) and pan the entire scene, potentially disrupting the experience or model positioning. Disabling panning ensures a more controlled and consistent interaction with your models.

But it is just something you might want to consider and maybe not ! 

Have a good one , and great job ! ... I tinkered with this React project ,too hehe (wanted to code something and was inspired by it then created something super similar it is just so neat)...

**Comment by ladunjexa** [comment]
Good point, @AaronGulman, I didn't pay attention to it before, added 🙌 
https://m.youtube.com/watch?v=Ra-Om7UMSJc

**Comment by AaronGulman** [comment]
> Good point, @AaronGulman, I didn't pay attention to it before, added 🙌 https://m.youtube.com/watch?v=Ra-Om7UMSJc

Great ,glad to help out ! 
Also, the song made your reply that much better !
